src/component/types.ts

```diff
@@ -0,0 +1,30 @@
+export interface AppState {
+  total: string | null
+  next: string | null
+  operation: string | null
+}
+
+export type PartialAppState = {
+  total?: string
+  next?: string
+  operation?: string
+}
+
+export interface DisplayProps {
+  value: string
+}
+
+export interface clickHandlerFunction {
```

Comment:
For interfaces/types use PascalCase naming convention. 
Change the name, it not quite descriptive.

```suggestion
export interface ClickHandlerFunction {
```

Reply:
Oh, that camelCase name slipped out while creating the interfaces, fixing it now. About the name should I just remove the Function part as you mentioned above or should I came up with a completely new name?